mailing-lists.php

```diff
@@ -124,12 +124,25 @@
  <a href="http://marc.info/">Marc</a>.
 </p>
 
-<h2>Twitter</h2>
+<h2>Social Media</h2>
 <p>
- The PHP team maintains an official PHP.net account on twitter,
- <a href="https://twitter.com/official_php">@official_php</a>, for those
- interested in following news and other announcements from the PHP project.
+ The PHP team maintains official PHP.net accounts on several social media
+ platforms for those interested in following news and other announcements
+ from the PHP project.
 </p>
+<ul>
+  <li>
+   <a href="https://fosstodon.org/@php">@php@fosstodon.org</a> on the
+   Fediverse/Mastodon.
+  </li>
+  <li>
+   <a href="https://www.linkedin.com/company/phpnet/">PHP</a> on LinkedIn.
+  </li>
+  <li>
+   <a href="https://x.com/official_php">@official_php</a> on X (formerly
+   Twitter).
+  </li>
```

Comment:
Can we be consistent with the home page?

<img width="343" alt="image" src="https://github.com/user-attachments/assets/8003f087-e5c7-46e6-9bde-fd02483104b7">

```suggestion
  <li>
   <a href="https://x.com/official_php">@official_php</a> on X (formerly
   Twitter).
  </li>
  <li>
   <a href="https://fosstodon.org/@php">@php@fosstodon.org</a> on the
   Fediverse/Mastodon.
  </li>
```

Reply:
Sure, although I'd rather make it consistent by moving X below Fediverse on the front page in order to promote the open platform ahead of the other one.

Or we could just remove this from the mailing-lists page entirely, since they aren't mailing lists.

Reply:
yep, it appears to be deletable. 🤔️